rfcs/interop_2023.md

```diff
@@ -0,0 +1,84 @@
+  - Start involving PR teams. Write rough drafts of announcements. Begin creating dashboard.
+
+During 2023:
+
+- January 9 to January 20:
+  - Finalize test selection.
+  - Complete announcement plans. Prepare dashboard for launch.
+- January 23 to January 27: Launch.
+
+If necessary, additional changes to the test lists can be made throughout the year with the consensus of the team, see [updating the tests](#updating-the-tests).
+
+### Governance
+
+The [Interop 2023 team](https://github.com/orgs/web-platform-tests/teams/interop-2023) will manage the overall Interop 2023 effort. The team will be formed with the same initial membership as the [’22 team](https://github.com/orgs/web-platform-tests/teams/interop-2022).
+
+The team will meet regularly, with agenda issues created in the repo at least 48 hours in advance, and meeting minutes posted in the issue.
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no objections. Additionally, during the [proposal selection](#proposal-selection) process, the positions of implementers in each browser engine are used to determine whether a proposal will be accepted, to ensure that the proposal is on track to interoperability across browser engines.
+
+The Interop 2023 effort is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+The [WPT core team](https://github.com/orgs/web-platform-tests/teams/wpt-core-team/) is not expected to be involved beyond this RFC, but could in exceptional cases intervene via the RFC process to amend the governance.
+
+### Proposals
+
+During the proposals period (September 15 to October 15), proposals are submitted as issues in the [Interop 2023 repo](https://github.com/web-platform-tests/interop-2023). A proposal should be as specific as reasonably possible, but multiple proposals may be grouped later in the selection process.
+
+The following information should be provided:
+
+- Standard/specification for this feature
+- Current test results on wpt.fyi
+- How widely used is this feature? For example, [use counters](https://www.chromestatus.com/metrics/feature/popularity), [HTTP Archive](https://httparchive.org/) and [State of CSS](https://2021.stateofcss.com/en-US/features/) can help.
+- Is this area a problem for web developers? For example, survey data and browser bug counts can help.
+- Is this area a problem for users?
+
+A proposals template will be created to help with this.
+
+### Proposal selection
+
+Proposal selection (November 1 to November 30) will be done using a positions process similar to [Interop 2022](https://github.com/web-platform-tests/interop-2022/issues/38) but with some changes. The process is:
+
+- Each participating organization takes one of 3 positions on each proposal: support, neutral, or oppose.
+- A proposal is accepted if there is implementer support coming from at least 2 browser engines, and no opposition.
```

Comment:
This is formalizing what we did for Interop 2022 in https://github.com/web-platform-tests/interop-2022/issues/39#issue-1073181003

> I suggest defining consensus as support from at least two organizations contributing to different browser engines, and a lack of opposition from everyone else.

It's noteworthy that the outcome would have been the same if we only required support from one organization and no opposition. That's because we didn't have any proposals with 1 support and everyone else neutral. I think it's worth a discussion how we want to handle that case if it arises.